### PR TITLE
[#860] Issue/limit max length of product summary

### PR DESCRIPTION
**Pull request by vaszig**

### CODE REVIEW

File: src/open_inwoner/pdc/migrations/0045_auto_20230220_1628.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from django.db import migrations
+
+
+def truncate_product_summary(apps, schema_editor):
+    MyModel = apps.get_model("pdc", "Product")
+    for product in MyModel.objects.all():
+        product.summary = product.summary[:300]
+        product.save()
+
```
Comment (Bartvaderkin) [inline src/open_inwoner/pdc/migrations/0045_auto_20230220_1628.py:11]: This save()'s every Product in the database, even if the summary was not too long.

File: src/open_inwoner/pdc/tests/test_data_migrations.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from open_inwoner.utils.tests.test_migrations import TestMigrations
+
+
+class TruncateProductSummaryMigrationTests(TestMigrations):
+    app = "pdc"
+    migrate_from = "0044_question_category_or_product_null"
+    migrate_to = "0045_auto_20230220_1628"
```
Comment (Bartvaderkin) [inline src/open_inwoner/pdc/tests/test_data_migrations.py:7]: Shouldn't this test to 0046_ ?

**Review by Bartvaderkin — CHANGES_REQUESTED** [review]

File: src/open_inwoner/pdc/tests/test_data_migrations.py (hunk 2 above)
Comment (vaszig) [reply src/open_inwoner/pdc/tests/test_data_migrations.py:7]: I wanted to test that the summary is truncated, that's why I tested the initial (0044_) compared to the data migration (0045_). If I am wrong let me know please.

**Review by Bartvaderkin — APPROVED** [review]

**Comment by alextreme** [comment]
Fails due to conflicting migrations after merging in #486

**Comment by Bartvaderkin** [comment]
Merging this as it is green and I need to rebase on it and use as migration dependency.

**Pull request merged**